src/common/kube-helpers.ts

```diff
@@ -16,16 +16,50 @@ function resolveTilde(filePath: string) {
   return filePath;
 }
 
+function loadFromFile(file: string) {
+  return loadFromString(fse.readFileSync(file, 'utf8'));
+}
+
+function loadFromString(config: string) {
+  try {
+    const obj = yaml.safeLoad(config);
+
+    // Validate the context structure for invalid context objects
+    if ('contexts' in obj) {
+      for (const [i, value] of obj["contexts"].entries()) {
+        if (!value.name || !value.context.cluster || !value.context.user) {
+          logger.error(`loadFromString: context ${JSON.stringify(value)} in kubeconfig has an error so will not be made available to Lens`);
+          obj["contexts"].splice(i, 1);
+        }
+      }
+    }
+    return yaml.dump(obj);
+  } catch (err) {
+    const error = String(err);
```

Comment:
this isn't used?

Reply:
Have added usage of error string into message